scripts/complete_liquidity_provision.js

```diff
@@ -136,16 +136,12 @@ module.exports = async (callback) => {
       true
     )
 
-    console.log("4. Sending out orders")
-    await signAndSend(masterSafe, orderTransaction, argv.network)
+    console.log("4. Sending the order placing transaction to gnosis-safe interface, please sign this transaction first")
```

Comment:
I propose to change "sign" to "execute" here, since I think that it's actually fine to sign them in any order.